packages/salesforcedx-apex-replay-debugger/src/messages/i18n.ts

```diff
@@ -23,5 +23,14 @@ export const messages = {
   no_log_file_text:
     'The log file either is missing or does not have any log lines in it.',
   incorrect_log_levels_text:
-    'The log must be generated with log categories Apex code at the FINEST level and Visualforce at the FINER level.'
+    'The log must be generated with log categories Apex code at the FINEST level and Visualforce at the FINER level.',
+  unable_to_retrieve_org_info: 'Unable to retrieve OrgInfo',
+  heap_dump_error:
+    'Problems were encountered while retrieving heap dump information. Message=%s, ErrorCode=%s. Heap Dump Information=%s',
+  heap_dump_error_wrap_up_text:
+    'Problems were encountered while retrieving heap dump information. See the messages above this one for details. Detailed heap dump variable information is unavailable for the specified line or lines.',
+  fetching_heap_dump:
+    'Retrieving heap dump from server. Heap dump information: %s',
```

Comment:
This and the one below are new messages in your latest commit. Do you need Ruth to review?

Reply:
`heap_dump_error_wrap_up_text` is the message I reviewed with Jim before being out of the office for two days. I'll suggest a couple of minor changes to the other messages.